homeassistant/components/whirlpool/sensor.py

```diff
@@ -32,6 +32,7 @@
     "3": "50",
     "4": "100",
     "5": "active",
+    None: "unknown",
```

Comment:
Instead of having this workaround here, could you do TANK_FILL.get(..., "unknown") on line 123, since that is safe to call even with None

Reply:
Good call - I think we actually want to return None on line 123, then HA will give it unknown.  Just need it NOT to die when the attribute is not available :)